xpad.c

```diff
@@ -1212,6 +1212,79 @@ static int xpad_start_xbox_one(struct usb_xpad *xpad)
 	return retval;
 }
 
+static int xpad_start_xbox_360(struct usb_xpad *xpad)
+{
+	int status;
+
+	char *data = kzalloc(20, GFP_KERNEL);
+
+#define TIMEOUT 100
+
+	status = usb_control_msg(xpad->udev,
+		usb_rcvctrlpipe(xpad->udev, 0),
+		0x1, 0xc1,
```

Comment:
Consider using constants defined in uapi/linux/usb/ch9.h instead of magic values (0x1, 0xc1, etc.)

0x1 = USB_REQ_CLEAR_FETAURE
0xc1 = USB_DIR_IN | USB_TYPE_VENDOR | USB_RECIP_INTERFACE
0x100 = USB_DEVICE_REMOTE_WAKEUP